src/frontend/src/utils/flowResult.ts

```diff
@@ -20,9 +21,9 @@ export type LoginFlowError = {
 } & LoginError;
 
 export type LoginError = {
-  title: string;
-  message: string;
-  detail?: string;
+  title: TemplateElement;
+  message: TemplateElement;
+  detail?: TemplateElement;
```

Comment:
`TemplateElement`s can be anything (including html), do you mean to use this or would the lighter version (`DynamicKey`, specific to the copy) work?

Reply:
It needs to be at least `string | DynamicKey` (I think). I.e. still allow the previous version of the code. But I should be able restrict to that.